words/words.md

```diff
@@ -695,25 +695,25 @@ Lass uns abschließend zurückkehren zu...
     </div>
 </div>
 
-Das Flugzeug ist gesunken. Zusammengedrängt hocken wir in den Rettungsinseln. Es ist Zeit, Land zu suchen.[^dry_land]
+Das Flugzeug ist versunken. Wir hocken zusammengedrängt in den Rettungsbooten. Es ist Zeit, Festland zu suchen.[^dry_land]
 
 [^dry_land]: Festland-Metapher [von Marc Lipsitch und Yonatan Grad, auf STAT News](https://www.statnews.com/2020/04/01/navigating-covid-19-pandemic/)
 
-Teams aus Epidemiologinnen und Menschen in der Politik ([links](https://www.americanprogress.org/issues/healthcare/news/2020/04/03/482613/national-state-plan-end-coronavirus-crisis/), [rechts](https://www.aei.org/research-products/report/national-coronavirus-response-a-road-map-to-reopening/ ), und [überparteilich](https://ethics.harvard.edu/covid-roadmap)) haben einen Konsens darüber erzielt, wie wir COVID-19 schlagen können und gleichzeitig unser Leben *und* unsere Freiheiten schützen.
+Teams aus Epidemiologinnen und Epidemiologen und Menschen in der Politik ([links](https://www.americanprogress.org/issues/healthcare/news/2020/04/03/482613/national-state-plan-end-coronavirus-crisis/), [rechts](https://www.aei.org/research-products/report/national-coronavirus-response-a-road-map-to-reopening/ ), und [überparteilich](https://ethics.harvard.edu/covid-roadmap)) haben einen Konsens darüber erzielt, wie wir COVID-19 schlagen können und gleichzeitig unser Leben *und* unsere Freiheiten schützen.
 
 Hier ist die grobe Idee, mit einigen (weniger konsensfähigen) Backup-Plänen:
```

Comment:
Ganz am Anfang haben wir schon Mal eine Stelle gehabt wo dann "backup plans" als "Plan B" übersetzt wurde. Das passt dann jetzt hiermit nicht zusammen. Die Option mit "Plan B" geht hier aber nicht, weil der Plural hier zwingend ist.

Reply:
Das stimmt, er ist hier zwingend. Also entweder wir sind konsistent und nehmen am Anfang und am Schluss den Plural oder wir sind inkonsistent und nehmen oben den Singular ("Plan" und "Plan B") und hier, Vorschlag: "Alternativplänen"

Ich tendiere, aus Gründen der Lesbarkeit beim Einstieg in den Text (das finde ich wichtig, weil ich glaube, dass so mehr Menschen zum Weiterlesen animiert werden, die mit dem Thema in der Tiefe nicht so viel zu tun haben und mal schnell unterhalten werden wollen), zu Variante 1.